pkg/syncer/metadatasyncer.go

```diff
@@ -2640,3 +2655,139 @@ func initResizeReconciler(ctx context.Context, tkgClient clientset.Interface,
 	rc.Run(ctx, resizeWorkers)
 	return nil
 }
+
+// createStoragePolicyUsageCR creates StoragePolicyUsage CR with given parameters
+func createStoragePolicyUsageCR(ctx context.Context, quotaClient client.Client, name, namespace,
+	storagePolicyId, storageClassName, resourceKind, resourceApiGroup,
+	extensionName string) (*storagepolicyusagev1alpha1.StoragePolicyUsage, error) {
+	newUsageInstance := storagepolicyusagev1alpha1.StoragePolicyUsage{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       resourceKind,
+			APIVersion: "v1",
```

Comment:
v1 or v1alpha1 ? Can you confirm

Reply:
good catch! my bad, put resource kind &version here instead of group, version of StoragePolicyUsage by mistake.
Fixed now.